node/src/Unison/Runtime/JournaledSet.hs

```diff
@@ -0,0 +1,72 @@
+{-# Language DeriveGeneric #-}
```

Comment:
If this module isn't being used anywhere, just delete it. Also, for journaled data structures that we do have in the codebase, I want to move away from having the convenience methods, in favor of just using `Journal.update/updateAsync/updateNowFlushAsync` directly.